src/FSAL/FSAL_CORTXFS/conf/nfs_setup.sh

```diff
@@ -99,6 +100,35 @@ function motr_lib_init {
 	[ $? -ne 0 ] && die "Failed to Initialise motr_lib fs_meta index"
 }
 
+function prepare_pnfs_conf {
+IFS=',' read -ra ADDR <<< "$pNFS_DATA_SERVER"
+count=0
+rm -rf /etc/ganesha/pNFS.conf
+echo "[pNFS]" >> /etc/ganesha/pNFS.conf
+echo "  pNFS_Role = $pNFS_ROLE" >> /etc/ganesha/pNFS.conf
+
+if [ "$pNFS_ROLE" != "PNFS_DISABLED" ]; then
+pNFS_ENABLED=true;
```

Comment:
Do we need to store pNFS_ENABLED value in pNFS.conf file ?

Reply:
No, this flag is just for Set_attr_expiration time parameter